server/src/pages/shot/view.js

```diff
@@ -212,7 +212,7 @@ class Body extends React.Component {
         <div>
           <div className="spacer"/>
           If you do nothing,<br/>
-          this shot will be permanently deleted <TimeDiff date={deleteTime} />.
+          this shot will be permanently deleted in <TimeDiff date={deleteTime} />.
```

Comment:
yeah, that's kind of why i pulled the verb`in` out so each of those strings is only a unit of time and localizers can do whatever with the antecedent string